sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/DateTimeUtilsSuite.scala

```diff
@@ -555,12 +555,12 @@ class DateTimeUtilsSuite extends SparkFunSuite with Matchers {
 
     // There are some days are skipped entirely in some timezone, skip them here.
     val skipped_days = Map[String, Set[Int]](
-      "Kwajalein" -> Set(8632, 8633),
+      "Kwajalein" -> Set(8632, 8633, 8634),
```

Comment:
Could you put the JDK / OS version for this into the PR description, please?

Reply:
I updated the JDK version. I'll ask the reporter to comment here with more details. That said, I think this is an obvious fix so I'm not too worried about those details. It doesn't happen on 1.8.0_222 at least.

Reply:
CC @darshan-pandya

Reply:
When we encountered, this test failed on jdk 1.8.0_232

Reply:
And, OS?

Reply:
I assume it's Linux @darshan-pandya but would it matter? Java uses its own TZ info rather than anything from the OS.

Reply:
Just out of curiosity, 😄 .